src/emulator/remoteconfig/cloudFunctions.ts

```diff
@@ -0,0 +1,123 @@
+import * as uuid from "uuid";
+import { EmulatorLogger } from "../emulatorLogger";
+import { Emulators } from "../types";
+import { Client } from "../../apiv2";
+import { EmulatorRegistry } from "../registry";
+import { toSerializedDate } from "../storage/metadata";
+import { CloudEvent } from "../events/types";
+import { RemoteConfigEventData } from "@google/events/firebase/remoteconfig/v1/RemoteConfigEventData";
+
+type RemoteConfigFunctionAction = "update";
+const REMOTE_CONFIG_V2_ACTION_MAP: Record<RemoteConfigFunctionAction, string> = {
+  update: "updated",
+};
+
+export interface RemoteConfigEventPayload {
+  description: string;
+  updateOrigin: string;
+  updateTime: string;
+  updateType: string;
+  updateUser: any;
+  versionNumber: number;
+}
+
+export class RemoteConfigCloudFunctions {
```

Comment:
Nit: This name could be a bit clearer. IIUC, this class mainly handles event dispatching - maybe RemoteConfigEventDispatcher?